README.md

```diff
@@ -187,59 +188,60 @@ data class Responses(
 
 This section documents the available CLI parameters for controlling what gets generated. This documentation is generated using: `./gradlew printCodeGenUsage`
 
-| Parameter                     | Description |
```

Comment:
Has this been regenerated using  `./gradlew printCodeGenUsage`?

Reply:
It has now ✨🙂